src/test/transaction_tests.cpp

```diff
@@ -104,10 +169,9 @@ BOOST_AUTO_TEST_CASE(tx_valid)
     // or [[[prevout hash, prevout index, prevout scriptPubKey], [input 2], ...],"], serializedTransaction, verifyFlags
     // ... where all scripts are stringified scripts.
     //
-    // verifyFlags is a comma separated list of script verification flags to apply, or "NONE"
+    // verifyFlags is a comma separated list of MINIMAL script verification flags to exclude, or "NONE"
```

Comment:
I don't think the format of tx_valid.json should be documented here _and_ in the file itself. It'd be better to consolidate this documentation to just the json file.

No need to capitalize MINIMAL.

Reply:
consolidated in the json since that's where any new tests would be added